src/components/Sum.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({
```

Comment:
you should be able to export right here
```suggestion
export const Sum = ({
```
then you don't seem to need export at last line